### PR TITLE
importccl: add decompression during schema reading too

### PR DESCRIPTION
**Pull request by dt**
We added support for compressed IMPORT input but only during the row-reading.
With the addition of the schema-reading pass, we need to use the same decompression there too.

This also checks in bzip and gzip testdata examples for MySQL so that we can test these in unit tests more easily.

Release note: none.

### CODE REVIEW

**Review by madelynnblue — APPROVED** [review]

File: pkg/ccl/importccl/pg_testdata_helpers_test.go [hunk 1]
```diff
@@ -92,6 +92,13 @@ func getMultiTablePostgresDumpTestdata(t *testing.T) string {
 			})
 		})
 	}
+	_ = os.Remove(dest + ".bz2")
+	out, err := exec.Command("bzip2", "-k", dest).CombinedOutput()
+	if err != nil {
+		t.Fatal(err, string(out))
+	}
+	gzipFile(t, dest)
```
Comment (mjibson) [inline pkg/ccl/importccl/pg_testdata_helpers_test.go]: Do these files need to be generated, checked in, and tested too?

Comment (dt) [reply pkg/ccl/importccl/pg_testdata_helpers_test.go]: it wouldn't hurt, but in both cases the code is outside the format-specific block.

Comment (dt) [reply pkg/ccl/importccl/pg_testdata_helpers_test.go]: oh, I meant to remove this part of the diff.

Comment (dt) [reply pkg/ccl/importccl/pg_testdata_helpers_test.go]: Fixed.

**Review by madelynnblue — APPROVED** [review]

**Pull request merged**